plugins/v1/repository_gcs/README.md

```diff
@@ -0,0 +1,28 @@
+This directory contains the (optional) keystore configuration for the `repository-gcs` plugin.
+For more details on secure settings for the repository-gcs plugin please refer to the [repository-gcs-client](https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-gcs-client.html) documentation.
+
+### Parameters
+
+This configuration allows to set the following parameters with Rally using `--plugin-params` in combination with `--elasticsearch-plugins="repository-gcs"`:
+
+* `gcs_credentials_file`: A string specifying the full path to the service account json file.
```

Comment:
Should we match the documentation style of the S3 plugin below and mention for each parameter that it is mandatory?

Reply:
Fixed in db748c6098d76529876ef5deac3eb35c73c9a4a0